error.php

```diff
@@ -21,11 +21,14 @@
     $tpl->setContent($local_tpl->get());
     $tpl->printToStdout();
 } catch (Exception $e) {
+    header("HTTP/1.1 500 Internal Server Error");
```

Comment:
(Nowadays) I suggest using the `$DIC['http`]` service to prepare and finally send the response.

Reply:
In general I would agree to use the $DIC, but I think that if we have already hit the error handler and are even failing there, I would not rely on having any kind of $DIC, etc available. I would be fine to merge this PR.